discover_overlay/locales/fr/LC_MESSAGES/default.po

```diff
@@ -187,7 +187,7 @@ msgid "Close overlay"
 msgstr "Fermer l’overlay"
 
 msgid "Overview"
-msgstr "Aperçu"
+msgstr "Informations"
```

Comment:
This might match the actual content better, but it derives from the source string, so I don’t think its relevant.

Reply:
I'd be happy for this to actually be 'About' or 'Information' as overview is considerably vague